mono/metadata/marshal.c

```diff
@@ -8120,14 +8123,12 @@ mono_marshal_get_native_wrapper (MonoMethod *method, gboolean check_exceptions,
 				case ICALL_HANDLES_WRAP_OBJ:
 					/* argI = mono_handle_new_full (argI_raw, FALSE) */
```

Comment:
Could you fixup this comment (and the ones in the other two cases) too? Other than that, LGTM.